packages/hw-app-btc/src/Btc.js

```diff
@@ -492,30 +504,41 @@ btc.createPaymentTransactionNew(
  ).then(res => ...);
    */
   createPaymentTransactionNew(
-    inputs: Array<[Transaction, number, ?string, ?number]>,
-    associatedKeysets: string[],
-    changePath?: string,
-    outputScriptHex: string,
-    lockTime?: number = DEFAULT_LOCKTIME,
-    sigHashType?: number = SIGHASH_ALL,
-    segwit?: boolean = false,
-    initialTimestamp?: number,
-    additionals?: Array<string>
+    {
+      inputs,
+      associatedKeysets,
+      changePath,
+      outputScriptHex,
+      lockTime = DEFAULT_LOCKTIME,
+      sigHashType = SIGHASH_ALL,
+      segwit = false,
+      initialTimestamp,
+      bip143 = false,
+      overwinter
+    }: {
+      inputs: Array<[Transaction, number, ?string, ?number]>,
+      associatedKeysets: string[],
+      changePath?: string,
+      outputScriptHex: string,
+      lockTime?: number,
+      sigHashType?: number,
+      segwit?: boolean,
+      initialTimestamp?: number,
+      bip143?: boolean,
+      overwinter?: {
+        expiryHeight: Buffer
+      }
+    }
   ) {
     const hasTimestamp = initialTimestamp !== undefined;
     let startTime = Date.now();
-    let useBip143 =
-      segwit ||
-      (!!additionals &&
-        (additionals.includes("abc") ||
-          additionals.includes("gold") ||
-          additionals.includes("bip143")));
+    let useBip143 = segwit || bip143 || overwinter;
     // Inputs are provided as arrays of [transaction, output_index, optional redeem script, optional sequence]
     // associatedKeysets are provided as arrays of [path]
     const nullScript = Buffer.alloc(0);
     const nullPrevout = Buffer.alloc(0);
-    const defaultVersion = Buffer.alloc(4);
-    defaultVersion.writeUInt32LE(1, 0);
+    let defaultVersion = Buffer.alloc(4);
+    overwinter ? defaultVersion.writeUInt32LE(2147483651, 0) : defaultVersion.writeUInt32LE(1, 0);
```

Comment:
maybe set a global constant for readability
`0x80000003` maybe more readable

Reply:
🙀 Yes, please avoid decimal for this kind of constants where 0x80000003 actually has a meaning.